server/blob/blob-api/src/test/java/org/apache/james/blob/api/MetricableBlobStoreContract.java

```diff
@@ -79,32 +57,43 @@ private void setupExpectations() {
 
     @Test
     default void saveBytesShouldPublishSaveBytesTimerMetrics() {
+        long sleepDurationInMs = Duration.ofSeconds(1).toMillis();
```

Comment:
Can we get smaller sleeps? 100 ms should be enough but will be fast enough to not annoy us IMO